lib/login_gov/hostdata/railtie.rb

```diff
@@ -0,0 +1,42 @@
+begin
+  require 'lograge'
+rescue LoadError => e
+  STDERR.puts 'lograge is required by the LoginGov::Hostdata::Railtie, please add it to the Gemfile'
+  raise e
+end
+
+if Gem::Version.new(Rails::VERSION::STRING) < Gem::Version.new('6.1')
+  raise 'LoginGov::Hostdata::Railtie needs rails >= 6.1 to log request info, please upgrade'
+end
+
+require 'login_gov/hostdata/log_formatter'
+require 'securerandom'
+
+module LoginGov
+  module Hostdata
+    class Railtie < Rails::Railtie
+      config.lograge.enabled = true
+
+      config.log_formatter = if Rails.env.development?
+        LoginGov::Hostdata::DevelopmentLogFormatter
+      else
+        LoginGov::Hostdata::LogFormatter
+      end
+
+      if Rails.env.development? || Rails.env.production?
+        config.lograge.formatter = Lograge::Formatters::Json.new
```

Comment:
Q: Is there a reason we don't want to use the JSON formatter for test environment? (to the point of: Does this need to be conditional?)

Reply:
This was copying the IDP's behavior 🤷 but I guess it seems like we could just bring it over